### ISSUE TITLE
Empty 'd' attribute values not allowed

### ISSUE THREAD

**Issue by ryanmccauley211** [Unity-Technologies/vector-graphics-samples#9]
When the d attribute value is empty it is treated as invalid, however according to W3C it should be not treated as an error:

d = "<path-data>"
The definition of the outline of a shape. See Path data. **An empty attribute value (d="") disables rendering of the element**. The lacuna value is the empty string.

see - https://www.w3.org/TR/SVGTiny12/paths.html#PathElement

I am using using poppler to convert pdf to svg which generates some empty d attributes such as:

```
<symbol overflow="visible" id="glyph6-14">
<path style="stroke:none;" d=""/>
</symbol>
```

I get the following error:

```
SVGFormatException: SVG Error (line 6, character 2): 'path' element missing 'd' attribute
Unity.VectorGraphics.SVGAttribParser.ParsePath (Unity.VectorGraphics.Node node) (at /Users/ryan/Library/Unity/cache/packages/staging-packages.unity.com/com.unity.vectorgraphics@1.0.0-preview.7/Runtime/SVGParser.cs:1969)
Unity.VectorGraphics.SVGDocument.path () (at /Users/ryan/Library/Unity/cache/packages/staging-packages.unity.com/com.unity.vectorgraphics@1.0.0-preview.7/Runtime/SVGParser.cs:509)
Unity.VectorGraphics.SVGDocument.ParseChildren (Unity.VectorGraphics.Node node, System.String nodeName) (at /Users/ryan/Library/Unity/cache/packages/staging-packages.unity.com/com.unity.vectorgraphics@1.0.0-preview.7/Runtime/SVGParser.cs:224)
Unity.VectorGraphics.SVGDocument.symbol () (at /Users/ryan/Library/Unity/cache/packages/staging-packages.unity.com/com.unity.vectorgraphics@1.0.0-preview.7/Runtime/SVGParser.cs:1013)
Unity.VectorGraphics.SVGDocument.ParseChildren (Unity.VectorGraphics.Node node, System.String nodeName) (at /Users/ryan/Library/Unity/cache/packages/staging-packages.unity.com/com.unity.vectorgraphics@1.0.0-preview.7/Runtime/SVGParser.cs:224)
Unity.VectorGraphics.SVGDocument.ParseChildren (Unity.VectorGraphics.Node node, System.String nodeName) (at /Users/ryan/Library/Unity/cache/packages/staging-packages.unity.com/com.unity.vectorgraphics@1.0.0-preview.7/Runtime/SVGParser.cs:225)
Unity.VectorGraphics.SVGDocument.defs () (at /Users/ryan/Library/Unity/cache/packages/staging-packages.unity.com/com.unity.vectorgraphics@1.0.0-preview.7/Runtime/SVGParser.cs:277)
Unity.VectorGraphics.SVGDocument.ParseChildren (Unity.VectorGraphics.Node node, System.String nodeName) (at /Users/ryan/Library/Unity/cache/packages/staging-packages.unity.com/com.unity.vectorgraphics@1.0.0-preview.7/Runtime/SVGParser.cs:224)
Unity.VectorGraphics.SVGDocument.svg () (at /Users/ryan/Library/Unity/cache/packages/staging-packages.unity.com/com.unity.vectorgraphics@1.0.0-preview.7/Runtime/SVGParser.cs:988)
Unity.VectorGraphics.SVGDocument.Import () (at /Users/ryan/Library/Unity/cache/packages/staging-packages.unity.com/com.unity.vectorgraphics@1.0.0-preview.7/Runtime/SVGParser.cs:182)
Unity.VectorGraphics.SVGParser.ImportSVG (System.IO.TextReader textReader, Single dpi, Single pixelsPerUnit, Int32 windowWidth, Int32 windowHeight) (at /Users/ryan/Library/Unity/cache/packages/staging-packages.unity.com/com.unity.vectorgraphics@1.0.0-preview.7/Runtime/SVGParser.cs:55)
Unity.VectorGraphics.Editor.SVGImporter.OnImportAsset (UnityEditor.Experimental.AssetImporters.AssetImportContext ctx) (at /Users/ryan/Library/Unity/cache/packages/staging-packages.unity.com/com.unity.vectorgraphics@1.0.0-preview.7/Editor/SVGImporter.cs:85)
UnityEditor.Experimental.AssetImporters.ScriptedImporter.GenerateAssetData (UnityEditor.Experimental.AssetImporters.AssetImportContext ctx) (at /Users/builduser/buildslave/unity/build/Editor/Mono/AssetPipeline/ScriptedImporter.cs:19)
```

**Comment by mcoted** [comment]
Thank you! This should be fixed in the next release.

**Comment by mcoted** [comment]
This should be fixed with version 1.0.0-preview.19
